### ISSUE TITLE
Mismatch in prometheusRules mixin for lvmLocalPV 

### ISSUE THREAD

**Issue by luisico** [openebs/monitoring#98]
As per the title, the mismatch is due to case difference in "lvmLocalPV" between these two files:

- `jsonnet/openebs-mixin/rules/prometheus-rules.libsonnet#L26` (lowercase):
   https://github.com/openebs/monitoring/blob/87b902c85429eedce78033049de9280f39e81372/jsonnet/openebs-mixin/rules/prometheus-rules.libsonnet#L26

- `jsonnet/openebs-mixin/rules/openebs/lvmlocalpv-rules.libsonnet#5` (mixed case):
   https://github.com/openebs/monitoring/blob/87b902c85429eedce78033049de9280f39e81372/jsonnet/openebs-mixin/rules/openebs/lvmlocalpv-rules.libsonnet#L5

This prevents compilation, as showed here:
```
RUNTIME ERROR: Field does not exist: lvmlocalpv
        vendor/openebs-mixin/rules/prometheus-rules.libsonnet:26:113-175        object <anonymous>
        vendor/openebs-mixin/mixin.libsonnet:9:20-62    object <anonymous>
        vendor/kube-prometheus/lib/mixin.libsonnet:24:78-109
        vendor/kube-prometheus/lib/mixin.libsonnet:26:19-20     object <anonymous>
        During manifestation
```
Changing to mixcase in `jsonnet/openebs-mixin/rules/prometheus-rules.libsonnet#L26` fixes the issue for me, i.e.
```diff
- ... + lvmLocalPV(prometheusRules._config).prometheusRules.lvmlocalpv + ...
+ ... + lvmLocalPV(prometheusRules._config).prometheusRules.lvmLocalPV + ...
```

**Comment by Quarky9** [comment]
Ran into same issue. Above fixed it.

**Comment by r1jt** [comment]
Closing the issue as the changes have been merged with PR #104
